### PR TITLE
RES-1615 translate more notifications

### PR DESCRIPTION
**Pull request by edwh**
Translate and test more notifications.

### CODE REVIEW

File: app/Http/Controllers/PartyController.php [hunk 1]
```diff
@@ -777,7 +777,7 @@ public function notifyHostsOfRsvp($user_event, $event_id)
             $hosts = null;
         }
 
-        if (! is_null($hosts)) {
+        if (count($hosts)) {
```
Comment (ngm) [inline app/Http/Controllers/PartyController.php:780]: I'm not sure how Laravel in a production environment if this is null - is it OK?  

Recent PHPs will give a warning if reporting is E_ALL.  https://3v4l.org/bZCgq
Laravel doesn't like warnings in production.

File: app/Notifications/JoinGroup.php [hunk 2]
```diff
@@ -74,19 +83,23 @@ public function toMail($notifiable)
         } else { // users not yet on the platform
```
Comment (ngm) [inline app/Notifications/JoinGroup.php:83]: This one is an interesting one - it is being sent to users not currently on the platform.  So not sure what `notifiable->language` will be.  We should probably create a separate ticket for this specific notification to think about how to deal with this (and the one for inviting to events, where presumably the same could happen).  We could maybe take the locale from the network the group is in (but then, what if it's in two networks etc etc...)

File: app/Notifications/ReviewNotes.php [hunk 3]
```diff
@@ -52,12 +52,16 @@ public function via($notifiable)
     public function toMail($notifiable)
     {
         if ($notifiable !== null && $notifiable->invites == 1) {
+            $locale = $notifiable->language;
+
             return (new MailMessage)
                   ->subject('A repair has been marked as suitable for the wiki')
```
Comment (ngm) [inline app/Notifications/ReviewNotes.php:58]: A couple of lines here don't seem to be going through translations.

**Review by ngm — CHANGES_REQUESTED** [review]
Looks great, just:

- a question about whether removing null check will be OK on prod
- a couple of places to create follow up tickets (I'll do these)
- a couple of lines in one of the notifications that don't seem to be made translateable

File: app/Http/Controllers/PartyController.php (hunk 1 above)
Comment (edwh) [reply app/Http/Controllers/PartyController.php:780]: Thanks - will make it robust against that.

File: app/Notifications/ReviewNotes.php (hunk 3 above)
Comment (edwh) [reply app/Notifications/ReviewNotes.php:58]: ReviewNotes is only sent to admins, though, right?  The aim for this story is only to have done all the non-admin ones.

Comment (ngm) [reply app/Notifications/ReviewNotes.php:58]: Oh yes, that's fine then.

**Review by ngm — APPROVED** [review]

File: app/Notifications/ReviewNotes.php (hunk 3 above)
Comment (edwh) [reply app/Notifications/ReviewNotes.php:58]: Actually it seems to have been removed in another branch; I think James decided he didn't need it.

File: app/Notifications/JoinGroup.php (hunk 2 above)
Comment (ngm) [reply app/Notifications/JoinGroup.php:83]: Created https://therestartproject.atlassian.net/browse/RES-1625

File: app/Notifications/NotifyHostRSVPInvitesMade.php [hunk 4]
```diff
@@ -46,24 +46,28 @@ public function via($notifiable)
      */
     public function toMail($notifiable)
```
[the remark this answers is not in the document]
Comment (ngm) [reply app/Notifications/NotifyHostRSVPInvitesMade.php:47]: Created https://therestartproject.atlassian.net/browse/RES-1626 for us to look at it.

**Pull request merged**